script.js

```diff
@@ -0,0 +1,53 @@
+console.log('script is running');
+
+alert('Welcome to RoseWood, can you keep a Secret?');
+
+var Liars = prompt("Help us find expose 'A' before it's too late! Type 'ssh' to continue!");
+  if ('ssh'){
+    console.log('Enter');
+  }
+
+alert('A' + '  '+ 'has discovered you\'re helping the liars and has uncovered a secret about you!');
+
+var A = prompt("I know what you did last summer, don't worry, I'm saving your later!" + "Do you want to continue? " + "Type 'Continue' to help the Liars");
+ if ("continue"){
+   console.log("continue");
+ }
+
+var Hanna = alert('thank-you for sticking with us, we know it\'s tough but soon we will find Ali');
+
+var Emily = alert ('WATCH OUT!' + '  '+ 'A' + ' is right behind you!');
+
+var action='running';
+  switch (action) {
+    case 'trip':
+      console.log('you\'ve fallen, hit'+ 'j' + 'to get back up!');
+  break;
+}
+  var Aria = prompt('take my hand!');
+    switch(action) {
+      case 'jumps to feet':
+       console.log('A' + ' ' + 'is gaining on you, HURRY!!');
+      break;
+     case 'A  jumps out of nowhere!':
+      console.log('cornered by A' + 'fight A to avoid going to Dollhouse');
+    break;
+      case 'high jump kick to stomach':
+        console.log('Liars join in');
+    break;
+      case 'A  is outnumbered!':
+       console.log('A  flees from site!');
+    break;
+      default:
+      console.log('A wins battle');
+    break;
+    }
+
+// function fight(){
+//       var action= 0;
+//        if( action === 'high jump kick to stomach') || (action === 'A is outnumbered'){
+//         action=10 points;
+//     }
+
+var Spencer= alert('way to help us get rid of A  that was a close one!');
+var Hanna= alert('Guys look, A left behind a clue!');
```

Comment:
Great effort overall, but there is still some work to be done with working with conditionals and control flow.